diff --git a/vendor/du/config/packages.mk b/vendor/du/config/packages.mk
index 6b805d8..e576a14 100644
--- a/vendor/du/config/packages.mk
+++ b/vendor/du/config/packages.mk
@@ -10,15 +10,10 @@ PRODUCT_PACKAGES += \
 
 # Optional packages
 PRODUCT_PACKAGES += \
-    Basic \
-    CalendarWidget \
-    Chromium \
-    DU-About \
-    DUCertified \
     LatinIME \
     LockClock \
-    OmniSwitch \
-    PhaseBeam
+    ViPER4Android \
+    OmniSwitch
 
 # Extra tools
 PRODUCT_PACKAGES += \
