diff --git a/vendor/du/config/common.mk b/vendor/du/config/common.mk
--- a/vendor/du/config/common.mk
+++ b/vendor/du/config/common.mk
@@ -81,13 +81,10 @@
 # Optional packages
 PRODUCT_PACKAGES += \
     Basic \
-    PhaseBeam \
-    Chromium \
-    DUCertified
+    PhaseBeam
 
 # Extra Optional packages
 PRODUCT_PACKAGES += \
-    DU-About \
     LatinIME \
     BluetoothExt \
     CalendarWidget \
@@ -135,16 +132,9 @@
 # Versioning System
 ANDROID_VERSION = 6.0.1
 DU_VERSION = v10.0
-ifndef DU_BUILD_TYPE
-    DU_BUILD_TYPE := DIRTY-DEEDS
-    PLATFORM_VERSION_CODENAME := DIRTY-DEEDS
-endif
+DU_BUILD_TYPE := DIRTY-Amy07i
+PLATFORM_VERSION_CODENAME := DIRTY-Amy07i
 
-#Build DU-Updater only if DU_BUILD_TYPE isn't DIRTY-DEEDS
-ifneq ($(DU_BUILD_TYPE),DIRTY-DEEDS)
-PRODUCT_PACKAGES += \
-    DU-Updater
-endif
 
 # Set all versions
 DU_VERSION := DU_$(DU_BUILD)_$(ANDROID_VERSION)_$(shell date -u +%Y%m%d-%H%M).$(DU_VERSION)-$(DU_BUILD_TYPE)
