diff --git a/build/target/product/core_minimal.mk b/build/target/product/core_minimal.mk
index 2e1ee76..7ef37c0 100644
--- a/build/target/product/core_minimal.mk
+++ b/build/target/product/core_minimal.mk
@@ -23,7 +23,6 @@ PRODUCT_DEVICE := generic
 PRODUCT_NAME := core
 
 PRODUCT_PACKAGES += \
-    aapt \
     BackupRestoreConfirmation \
     DownloadProvider \
     HTMLViewer \
@@ -77,6 +76,7 @@ PRODUCT_PACKAGES += \
     uncrypt \
     voip-common \
     webview \
+    WebViewGoogle \
     wifi-service
 
 PRODUCT_COPY_FILES += \
