diff --git a/build/target/product/core.mk b/build/target/product/core.mk
index 646f5bb..649a9fa 100644
--- a/build/target/product/core.mk
+++ b/build/target/product/core.mk
@@ -21,14 +21,10 @@
 # core_tiny.mk in addition to core_minimal.mk.
 
 PRODUCT_PACKAGES += \
-    BasicDreams \
     CalendarProvider \
     CaptivePortalLogin \
     CertInstaller \
     Contacts \
-    DeskClock \
-    DocumentsUI \
-    DownloadProviderUi \
     ExactCalculator \
     ExternalStorageProvider \
     FusedLocation \
